tests/functional/centos/7/cluster/Vagrantfile

```diff
@@ -1 +0,0 @@
-../../../../../Vagrantfile
```

Comment:
Are the removal of the files related to this commit which wants to 'play lvm_setup.yml on all scenarios'?

Reply:
no, they are related to the scenarios renaming